Translate plugins/eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay opened the pull request:
## Summary 

// #535 

## 리뷰 참고 사항

line19,29,30,31,32,34,38,60 소스 맵->소스맵
line44
다음 예시는 EvalSourceMapDevToolPlugin의 흔히 사용되는 사례를 보여줍니다.
->다음 예시는 EvalSourceMapDevToolPlugin이 흔히 사용되는 사례를 보여줍니다.

L27, L60 문장 끝 마침표 처리
기여자 작성

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Create eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Merge pull request #1 from nicekim2000/nicekim2000-patch-1

Create eval-source-map-dev-tool-plugin.mdx

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/line/webpack.kr?pullRequest=536) <br/>All committers have signed the CLA.

dkstyle commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -16,36 +16,36 @@ related:
 ## Examples
 
-The following examples demonstrate some common use cases for this plugin.
+다음 예시는 EvalSourceMapDevToolPlugin의 흔히 사용되는 사례를 보여줍니다.
```
다음 예시는 EvalSourceMapDevToolPlugin의 흔히 사용되는 사례를 보여줍니다.

아래 방식처럼 변경하면 조금더 자연스럽지 않을까 의견 드립니다.
-> 다음 예시는 EvalSourceMapDevToolPlugin이 흔히 사용되는 사례를 보여줍니다. 
-> 다음 예시는 EvalSourceMapDevToolPlugin의 일반 사용 사례를 보여줍니다.

jaehyuk-jay replied:
다음 예시는 EvalSourceMapDevToolPlugin이 흔히 사용되는 사례를 보여줍니다.
로 수정 완료하였습니다.

dkstyle commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -16,36 +16,36 @@ related:
 ---
 
-This plugin enables more fine grained control of source map generation. It is also enabled automatically by certain settings of the [`devtool`](/configuration/devtool/) configuration option.
+EvalSourceMapDevToolPlugin을 사용하면 소스 맵 생성을 보다 세밀하게 제어할 수 있습니다. 또한 [`devtool`](/configuration/devtool/) 설정 옵션의 특정한 세팅에 의해 자동으로 활성화됩니다.
```
기존 다른 문서에서 SourceMap을 `소스맵`으로 붙여서 번역하고 있는것 같아서 이부분을 전체적으로 일치 시키면 좋을 것 같습니다.
`소스 맵` -> `소스맵`

jaehyuk-jay replied:
소스 맵->소스맵 으로 통일시켰습니다.

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

line19,29,30,31,32,34,38,60 소스 맵->소스맵
line44 
다음 예시는 EvalSourceMapDevToolPlugin의 흔히 사용되는 사례를 보여줍니다.
->다음 예시는 EvalSourceMapDevToolPlugin이 흔히 사용되는 사례를 보여줍니다.

jaehyuk-jay committed:
Merge pull request #2 from nicekim2000/nicekim2000-patch-2

Update eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Merge branch 'line:kr' into kr

dkstyle reviewed (approved):
LGTM

keipark commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -57,7 +57,7 @@ module.exports = {
 ### Exclude Vendor Maps
 
-The following code would exclude source maps for any modules in the `vendor.js` bundle:
+다음 코드는 `vendor.js` 번들에 있는 모든 모듈에 대한 소스맵을 제외합니다:
```
문장 끝은 `.` 마침표 처리 부탁드립니다. (L27, L60)

jaehyuk-jay replied:
문장 끝 마침표 처리 완료하였습니다.

dkstyle replied:
`제외합니다:` -> `제외합니다.` 로 수정을 요청하신 것 같은데요. 
수정이 `제외합니다.:` 로 반영된 것 같습니다.

jaehyuk-jay replied:
참고하여 수정하였습니다 . 감사합니다.

keipark commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -16,36 +16,36 @@ related:
     url: https://survivejs.com/webpack/building/source-maps/#sourcemapdevtoolplugin-and-evalsourcemapdevtoolplugin
```
다른 작업 참고하셔서 기여자도 남겨주세요 😄

jaehyuk-jay replied:
참고하여 기여자 작성완료하였습니다.

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

L27, L60 문장 끝 마침표 처리 
기여자 작성

jaehyuk-jay committed:
Merge pull request #3 from nicekim2000/nicekim2000-patch-2

Update eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

기여자 작성수정

jaehyuk-jay committed:
Merge pull request #4 from nicekim2000/nicekim2000-patch-3

Update eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

오타 수정

jaehyuk-jay committed:
Merge pull request #5 from nicekim2000/nicekim2000-patch-4

Update eval-source-map-dev-tool-plugin.mdx

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

L29 , L62  .: -> . 으로 수정

YukJiSoo commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -11,41 +11,43 @@ contributors:
+- `test` (`string|RegExp|array`): 모듈의 확장자를 기반으로 하는 소스맵을 포함합니다(기본값은 `.js` 및 `.css`).
+- `include` (`string|RegExp|array`): 주어진 값과 일치하는 모듈 경로에 대한 소스맵을 포함합니다.
+- `exclude` (`string|RegExp|array`): 소스맵 생성에서 주어진 값과 일치하는 모듈을 제외합니다.
+- `append` (`string`): 원래 애셋에 주어진 값을 추가합니다. 보통 `#sourceMappingURL` 을 주석으로 합니다. `[url]`은 소스맵 파일의 URL로 대체됩니다. 'false'는 추가를 비활성화합니다.
```
' 을 ` 으로 바꿔주세요!

jaehyuk-jay replied:
참고하여 수정하였습니다 !

YukJiSoo commented on src/content/plugins/eval-source-map-dev-tool-plugin.mdx:
```diff
@@ -11,41 +11,43 @@ contributors:
+- `include` (`string|RegExp|array`): 주어진 값과 일치하는 모듈 경로에 대한 소스맵을 포함합니다.
+- `exclude` (`string|RegExp|array`): 소스맵 생성에서 주어진 값과 일치하는 모듈을 제외합니다.
+- `append` (`string`): 원래 애셋에 주어진 값을 추가합니다. 보통 `#sourceMappingURL` 을 주석으로 합니다. `[url]`은 소스맵 파일의 URL로 대체됩니다. 'false'는 추가를 비활성화합니다.
+- `moduleFilenameTemplate` (`string`):  [`output.devtoolModuleFilenameTemplate`](/configuration/output/#outputdevtoolmodulefilenametemplate)를 참조하세요.
```
를 -> 을

jaehyuk-jay replied:
수정하였습니다 !

jaehyuk-jay committed:
Update eval-source-map-dev-tool-plugin.mdx

L34 ' 을 ` 로 수정 하였습니다.
L35 을 -> 를 로 수정하였습니다.
